conversational-client/src/app/app.component.ts

```diff
@@ -7,13 +13,21 @@ import {debounce} from './util/debounce';
 import {HistoryTutorComponent} from '@components/agents/history-tutor/history-tutor.component';
 import {FakeAgentComponent} from '@components/agents/fake-agent/fake-agent.component';
 import {ActivatedRoute} from '@angular/router';
+import {Agent, AgentState} from '@data/conversation';
+
+// Conversational agent without UI or world state.
+class DefaultAgent implements Agent {
```

Comment:
how about pulling this into an agents directory for consistency?
I can imagine somebody wanting to use a template would look for a "default-agent-component" before they look inside app.component.

Reply:
Done - created a default agent component that exemplifies how to use inputs.